lib/kennel/tasks.rb

```diff
@@ -50,6 +50,15 @@
     Kennel::UnmutedAlerts.print(Kennel.send(:api), tag)
   end
 
+  desc "show monitors with no data by TAG, for example TAG=team:foo"
+  task nodata: :environment do
+    monitors = Kennel.send(:api).list("monitor", monitor_tags: tag, group_states: "no data", with_downtimes: "true")
```

Comment:
I think this needs to be smarter and say "with each group state being no-data" ... otherwise we get monitors were some things are no-data ?

Reply:
updated. did a select! on overall_state: 'no data'